bundles/org.openhab.ui/doc/components/src/generate.js

```diff
@@ -127,3 +134,5 @@ Object.keys(widgetLibraries).forEach((l) => {
 })
 
 fs.writeFileSync('../index.md', index)
+
```

Comment:
Please remove these 2 extra lines at the end.

Reply:
Done